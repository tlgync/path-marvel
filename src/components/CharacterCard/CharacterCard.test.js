import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { CharacterCard } from '.';

test('CharacterCard SnapShot', () => {
  const name = '3-D Man';
  const image = {
    path: 'http://i.annihil.us/u/prod/marvel/i/mg/c/e0/535fecbbb9784',
  };
  const id = 1011334;
  const tree = renderer.create((
    <Router>
      <CharacterCard name={name} image={image} id={id} />
    </Router>
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
