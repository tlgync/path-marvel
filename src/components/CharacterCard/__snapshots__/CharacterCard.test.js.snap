// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterCard SnapShot 1`] = `
<a
  href="/detail/1011334"
  onClick={[Function]}
  style={
    Object {
      "color": "rgba(0,0,0,.85)",
    }
  }
>
  <div
    className="character"
  >
    <img
      alt="character"
      className="character-img"
      src="http://i.annihil.us/u/prod/marvel/i/mg/c/e0/535fecbbb9784.jpg"
    />
    <div
      className="character-name"
    >
      3-D Man
    </div>
  </div>
</a>
`;
